packages/corvid-local-site/src/utils/wixFiles.js

```diff
@@ -1,12 +1,38 @@
 const { prettyStringify } = require("./prettify");
+const isString_ = require("lodash/isString");
+const flow_ = require("lodash/flow");
+const atob = require("atob");
```

Comment:
personally I wouldn't use an npm module but rather create a util function

Reply:
Care to explain why?
I believe the library does exactly what you would write on your own